### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mjawadbutt**
this is pr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # learn-github-actions
 learn-github-actions
+updated!
```
Comment (rehzon) [inline README.md:3]: remove this crap.

Comment (rehzon) [inline README.md:3]: this is start review.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # learn-github-actions
```
Comment (rehzon) [inline README.md:1]: this is on line 1.

**Review by rehzon — CHANGES_REQUESTED** [review]
finishing with request changes.

File: README.md (hunk 1 above)
Comment (rehzon) [reply README.md]: ok resolving

File: README.md [hunk 3]
```diff
@@ -1,2 +1,3 @@
-# learn-github-actions
-learn-github-actions
+# learn-github-actions updated!
+learn-github-actions.
+updated again.
```
Comment (rehzon) [inline README.md:3]: line 2 sucks

Comment (rehzon) [inline README.md:3]: line 3 single comment.

**Review by rehzon — COMMENTED** [review]
general feedback without explicit approval.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,3 @@
-# learn-github-actions
-learn-github-actions
+# learn-github-actions updated!
```
Comment (rehzon) [inline README.md:1]: add single comment again (line 1)

File: README.md [hunk 5]
```diff
@@ -1,2 +1,3 @@
-# learn-github-actions
-learn-github-actions
```
Comment (rehzon) [inline README.md:2]: add comment using start a review button.

**Comment by rehzon** [comment]
cnbnbv

**Comment by rehzon** [comment]
vbnvb

**Review by rehzon — COMMENTED** [review]
Submit general feedback without explicit approval. 2222

**Review by rehzon — APPROVED** [review]
Submit feedback and approve merging these changes.

**Review by rehzon — CHANGES_REQUESTED** [review]
Submit feedback that must be addressed before merging.

**Review by rehzon — APPROVED** [review]
mnvbmbnmbn

File: README.md [hunk 6]
```diff
@@ -1,2 +1,3 @@
-# learn-github-actions
```
Comment (rehzon) [inline README.md:1]: why deleted line 1

Comment (rehzon) [reply README.md:1]: this is reply

File: README.md (hunk 4 above)
Comment (rehzon) [inline README.md:1]: why add this line

**Review by rehzon — COMMENTED** [review]
dfgfdg

File: README.md [hunk 7]
```diff
@@ -1,2 +1,3 @@
...
-learn-github-actions
+# learn-github-actions updated!
+learn-github-actions.
```
Comment (rehzon) [inline README.md:2]: dfgdfgf

Comment (rehzon) [reply README.md:2]: kjlhjklhjk

Comment (rehzon) [reply README.md:2]: eiuoyroetyht

File: README.md (hunk 3 above)
Comment (rehzon) [inline README.md:3]: mncbv,cvbvc

Comment (rehzon) [reply README.md:3]: under same

Comment (rehzon) [inline README.md:3]: starting new review?

**Review by rehzon — CHANGES_REQUESTED** [review]
after starting new review the textbox

**Review by rehzon — DISMISSED** [review]
appriving

**Review by rehzon — APPROVED** [review]
approving

**Comment by mjawadbutt** [comment]
closing

**Pull request merged**